backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1957,3 +1957,13 @@ databaseChangeLog:
           - addPrimaryKey:
               tableName: spring_session_attributes
               columnNames: session_primary_id, attribute_name
+  - changeSet:
+        id: alter-spring-session-types
+        author: emma@skylight.digital
+        comment: Change the creation_time, last_access_time, and expiry_time columns to be of type bigint instead of DATETIME. Spring expects bigints for these columns.
+        changes:
+          - sql:
+            sql: |
+              ALTER TABLE ${database.defaultSchemaName}.spring_session ALTER COLUMN creation_time TYPE bigint USING EXTRACT(EPOCH FROM creation_time);
```

Comment:
I assume that there's no data in any instance of this table in any case, since spring session crashes rather than use it?

Reply:
Yeah, there's no data in any instances of the table. But it wasn't possible to cast the datetimes to bigint directly, so I had to do this EXTRACT EPOCH anyway.

Reply:
Thanks for the explanation! I was confused why we would bother with that.